.vscode/settings.json

```diff
@@ -2,5 +2,7 @@
   "jest.pathToJest": "npm run jest --",
   "editor.rulers": [
     80
-  ]
+],
+"eslint.enable": true,
+"javascript.validate.enable": false
```

Comment:
This is a big mistake. I get this committed with each PR I do, as this is part of my development workflow. Sorry! Will remove it.